.circleci/config.yml

```diff
@@ -2,7 +2,7 @@ version: 2
 jobs:
   build:
     docker:
-      - image: circleci/node:8.12
+      - image: circleci/node:12.14.1
```

Comment:
A nested dependency required >=10, so I figured I would upgrade to the latest LTS version.